server/integration_tests/test_data/e2e_triple/whatstrandorientationdoesfgfr1have/chart_config.json

```diff
@@ -27,7 +26,7 @@
                       "bio/mm10_Fgfr1",
                       "bio/mm9_Fgfr1"
                     ],
-                    "title": "The strandOrientation for Orientation, FGFR1 and 6 more other are as follows:",
+                    "title": "The strandOrientation for FGFR1, FGFR1 and 5 more other are as follows:",
```

Comment:
I think this should say other**s** (plural)

Reply:
doing title/messaging changes in a separate pr!